### PR TITLE
Cleaning up styles and output for the campaign content pages.

### PR DESCRIPTION
**Pull request by weerd**
This PR does some additional tweaks, responsive fixes, etc to wrap up the Campaign content pages for now, with the two hardcoded CTAs.

Refs https://trello.com/c/eCW7M75Z/394-3-as-a-developer-i-want-to-be-able-to-add-cta-blocks-to-the-faq-and-scholarship-pages-so-that-i-can-push-users-to-affiliate-or-t

### CODE REVIEW

File: resources/assets/components/Markdown/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import classnames from 'classnames';
 import { markdown } from "../../helpers";
 import './markdown.scss';
 
-const Markdown = ({className = 'markdown', children}) => (
+const Markdown = ({className = null, children}) => (
```
Comment (weerd) [inline resources/assets/components/Markdown/index.js]: We already have `markdown` always being output, so this was outputting `markdown` as a class twice. `classnames` will ignore outputting `null`, so this should be safe.

File: resources/assets/components/CallToActionBlock/cta.scss [hunk 2]
```diff
@@ -26,44 +32,51 @@
...
+  .cta__title {
+    font-size: $font-medium;
+    font-weight: $weight-sbold;
+  }
 
-.cta__message {
-  font-size: $font-regular;
-  font-weight: $weight-normal;
-}
+  .cta__message {
```
Comment (DFurnes) [inline resources/assets/components/CallToActionBlock/cta.scss]: Do these need to be nested? Not a big deal, but feels like a specificity problem waiting to happen.

File: resources/assets/components/Markdown/index.js (hunk 1 above)
Comment (DFurnes) [inline resources/assets/components/Markdown/index.js]: Good catch!

File: resources/assets/components/CallToActionBlock/cta.scss (hunk 2 above)
Comment (weerd) [inline resources/assets/components/CallToActionBlock/cta.scss]: Yep, the specificity problem came from Forge, so I had to nest them 😞

Comment (DFurnes) [inline resources/assets/components/CallToActionBlock/cta.scss]: Ahhh, makes total sense. 👍

**Pull request merged**